src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: updates/post
+title: "DevTools Digest: More power with the new Command Menu"
+published_on: 2016-04-22
+updated_on: 2016-04-22
+authors:
+  - pbakaus
+tags:
+  - devtools
+  - digest
+  - update
+description: "Read about DevTools' new command menu and its over 60 actions that enable crazy fast workflows."
+featured_image: /web/updates/images/2016/04/devtools-digest/command-menu.png
+---
+
+<p class="intro">Read about DevTools' new <strong>command menu</strong> and its over 60 actions that enable crazy fast workflows.</p>
+
+## CMD+Shift+P to bring up the Command Menu
+
+![Command Menu in DevTools](/web/updates/images/2016/04/devtools-digest/command-menu.png)
+
+The "Jump to File" dialog that appears when you press <kbd>CMD+P</kbd> in the Sources panel isn't terribly well known, but has been around for a while. We've now went much further than that and developed a text-editor-inspired command menu that can drive almost every important action in DevTools.
+
+Hit <kbd>CMD+Shift+P</kbd> anywhere (even when the page is focussed!) to bring up the Command Menu, then type to filter and hit return to trigger the action. A few sample actions you could try:
```

Comment:
A couple notes on keyboard references. These are all in relation to the devtools docs conventions: 
* `return` should be formatted as a key (e.g. `<kbd>Return</kbd>`). 
* I've been using `Enter`, not `return`. 
* Keys are usually camel case (e.g. `Cmd`, not `CMD`).
* The `+` sign is not usually formatted as code. E.g. in the docs we would write `<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>`